### ISSUE TITLE
Docs: packages/handbook-v1/en/declaration files/Introduction.md -- "This guide" in some places

### ISSUE THREAD

**Issue by elibarzilay** [microsoft/TypeScript-Website#632]
`packages/handbook-v1/en/declaration files/Introduction.md`, first paragraph:

"This guide" (here and below and probably many other places) sounds weird, since it gives me a feeling that the whole documentation thing is a patchwork of random texts.  (Whether it *is* one is a different question...)  So maybe rephrase them to "This page" or "We will learn how to..." etc?

**Comment by orta** [comment]
Agree, calling it a 'section' in an upcoming PR
